addons/hr_contract/static/src/js/hr_contract_history_list_controller.js

```diff
@@ -0,0 +1,22 @@
+/** @odoo-module **/
+
+import ListController from "web.ListController";
+
+export const HrContractHistoryListController = ListController.extend({
+
+    /**
+     * @override
+     */
+    _onCreateRecord(ev) {
+        if (ev) {
+            ev.stopPropagation();
+        }
+        this.do_action({
+            name: 'New Employee',
```

Comment:
Should use _lt() otherwise it will not be translated:

```suggestion
            name: _lt('New Employee'),
```